runtime/runtime/src/prefetch.rs

```diff
@@ -210,7 +221,11 @@ mod tests {
         test_utils::{create_test_store, test_populate_trie},
         ShardTries, ShardUId, Trie, TrieConfig,
     };
-    use std::{rc::Rc, str::FromStr, time::Duration};
+    use std::{
+        rc::Rc,
+        str::FromStr,
+        time::{Duration, Instant},
+    };
```

Comment:
We don't group imports like this in nearcore: https://near.github.io/nearcore/practices/style.html#import-granularity

Best to update your local rust analyzer setting, then it will be done automatically.
We are not very strict about this though, you will find lots of (mostly old) code that groups it different than the style guide suggests.

Reply:
ah, indeed, so I guess imports above also violate this, so I'm changing those as well